pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+# Pull Request Template
+
+## Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
```

Comment:
Kindly check options that are relevant

Reply:
I would argue in favor of something in the line of:

Please mark options that are relevant.